core/server/src/test/java/alluxio/master/journal/ProtoBufJournalFormatterTest.java

```diff
@@ -19,4 +29,18 @@
   protected JournalFormatter getFormatter() {
     return new ProtoBufJournalFormatter();
   }
+
+  @Test
+  public void truncatedEntryDoesntFail() throws Exception {
+    ByteArrayOutputStream baos = new ByteArrayOutputStream();
+    ProtoBufJournalFormatter formatter = new ProtoBufJournalFormatter();
+    formatter.serialize(JournalEntry.newBuilder().setCompleteFile(
```

Comment:
If the master is restarted, would the entries be appended to the existing journal?